uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -1201,6 +1201,36 @@ sub make_nii {
 }
 
 =pod
+Creates pics associated with MINC files
+=cut
+sub make_minc_pics {
+    my ($dbhr, $TarchiveSource, $profile, $minFileID) = @_;
+    my $where = "WHERE TarchiveSource = ? ";
+    my $query = "SELECT Min(FileID) AS min, Max(FileID) as max FROM files ";
+    $query    = $query . $where;
+    my $sth   = $${dbhr}->prepare($query);
+    $sth->execute($TarchiveSource);
+    print "TarchiveSource is " . $TarchiveSource . "\n";
+
+    my $script = undef;
+    my $output = undef;
+    my @row = $sth->fetchrow_array();
+    if (@row) {
+        $script = "mass_pic.pl -minFileID $row[$minFileID] -maxFileID $row[1] ".
+                     "-profile $profile";
+        print "Running mass_pic as follows : " .$script . "\n";
+
```

Comment:
@gluneau 

1) same comment re. "print" statement
2) original script had -verbose option appended to the $script command if $verbose is set (you will probably need to pass this variable along with the function call), so it is good to keep this so that it is consistent with what we had before